clowdapp.yaml

```diff
@@ -416,9 +416,6 @@ parameters:
   value: "true"
 - name: PUMA_MAX_THREADS
   value: "5"
-- name: POSTGRESQL_MAX_CONNECTIONS
-  description: Number of maximum database connections
-  value: "50"
```

Comment:
Yeah that's not used at all and thought it's worth removing it - is that an issue  ? ( I can separate that change in a different commit but otherwise, I don't see how fixing up stuff we see wrong while working on a PR can be an issue )

Reply:
Just checking :) I think it's fine to include.

Reply:
@Victoremepunto even if this isn't used, I'd rather not remove it here as it is a different concern. I'd suggest looking into why it is missing and why it was set... Anyhow, the default seems to be typically 100, as per documentation https://www.postgresql.org/docs/12/runtime-config-connection.html so removal should be fine (?).

Reply:
My guess is that this was added by error at some point - this `POSTGRESQL_MAX_CONNECTIONS` make sense on the PostgreSQL db container like can be seen [here](https://github.com/sclorg/postgresql-container/tree/generated/12#environment-variables-and-volumes) - we used this in [the CI, QA environments on the previous deployment templates](https://github.com/RedHatInsights/e2e-deploy/blob/ff96f1019828929453d33ad710ea688930d064d9/templates/compliance/compliance-db.yml#L106) , not anymore